docs/index.md

```diff
@@ -1,3 +1,43 @@
-## The UW MRO automated focusing system
+# The UW MRO automated focusing system
 
-These pages will describe the automated focusing system being developed by students in UW Astronomy Undergrad Engineering Group (AUEG).
+These pages will describe the automated focusing system being developed by students in UW Astronomy Undergrad Engineering Group (AUEG). We are writing in python. There are two ways to achieve focusing:  
+a) measure FWHM and take the smallest value  
+b) look for the brightest pixels in the image to calculate radius of star and go from there
+
+## Here is a list of steps to take, in order:
```

Comment:
Shorter header please. Maybe something like: `## Steps to build automated focuser`?